src/py_class/py_class_impl.py

```diff
@@ -877,7 +885,7 @@ def inplace_numeric_operator(special_name, slot):
     '__itruediv__': inplace_numeric_operator('nb_inplace_true_divide'),
     '__ifloordiv__': inplace_numeric_operator('nb_inplace_floor_divide'),
     '__imod__': inplace_numeric_operator('nb_inplace_remainder'),
-    '__ipow__': unimplemented(),
+    '__ipow__': inplace_numeric_operator('nb_inplace_power'),
```

Comment:
`nb_inplace_power` is a `ternaryfunc` unlike the other in-place operators.
So I think this won't work as-is (or at a minimum, needs a test case).